docs/docusaurus/sidebars.json

```diff
@@ -289,9 +289,68 @@
       "label": "Integrations",
       "link": {"type": "doc", "id": "integrations/integrations"},
       "items": [
-        "integrations/authenticate_to_aws_resources",
-        "integrations/authenticate_to_azure_storage",
-        "integrations/authenticate_to_gcs",
+        {
+          "type": "category",
+          "label": "Cloud Authentication",
+          "description": "Use AWS, Azure, or GCP authentication",
+          "link": {"type": "doc", "id": "cover_pages/csp_auth"},
+          "items": [
+            "integrations/authenticate_to_aws_resources",
+            "integrations/authenticate_to_azure_storage",
+            "integrations/authenticate_to_gcs"
+          ]
+        },
+        {
+          "type": "category",
+          "label": "Data Warehouse",
```

Comment:
maybe change Data Warehouse to Data Lake? we have the "Query Data Lakes" chapter

Reply:
updated